### DISCUSSION TITLE
Add sort order: merged

### DISCUSSION THREAD

**Discussion by dhardy** [community/community#145421] [Pull Requests]
### Select Topic Area

Product Feedback

### Body

Please add sort order "recently merged". This would correlate with the order of git commits and be an intuitive way to browse project history. It should probably be the default sort order (opinionated).

By default, pull-requests are ordered by the PR number (equivalent to most recently created). This is often useful, but can make it hard to find PRs which were merged a long time after having been opened.

It is also possible to sort by "recently updated". This has some usage, but is mostly worse than "recently merged" since for most purposes it is unimportant to see recent comments on and links to old PRs.

**Comment by bokdrol21** [comment]
Yes please!
